### ISSUE TITLE
Depends on a missing branch of ggez

### ISSUE THREAD

**Issue by jaredly** [icefoxen/ggj2017#1]
`cargo run` fails due to no `ggj2017` branch existing on ggez https://github.com/icefoxen/ggj2017/blob/master/Cargo.toml#L7

**Comment by icefoxen** [comment]
Yep, sorry.  That's what happens when you have to hack on your engine during a game jam.  :/

Once I have ggez 0.3 working (a month or so maybe hopefully???) I'll be porting this forward to that to see how it works out.

**Comment by jaredly** [comment]
haha sounds good
